h/static/scripts/annotator/plugin/toolbar.coffee

```diff
@@ -36,7 +39,7 @@ class Annotator.Plugin.Toolbar extends Annotator.Plugin
             @annotator.triggerHideFrame()
     ,
       "title": "Hide Highlights"
-      "class": "h-icon-visibility"
+      "class": "annotator-highlight-visibility h-icon-visibility"
```

Comment:
Consistency in class names would be appreciated. You've got `annotator-highlight-visibility` here but `annotator-toolbar-highlight-btn` below. I'd also prefer if you used a data attribute for these things since I don't really like making code semantics depend on CSS classes. As a designer, one should be able to change CSS classes to their liking without worrying it will break code.